api/ipsec/validate

```diff
@@ -101,6 +101,10 @@ $v->declareParameter('pfs', $yn);
 $v->declareParameter('compress', $yn);
 $v->declareParameter('dpdaction', $yn);
 
+if( preg_match("/ /", $data['name'])) {
```

Comment:
You could use `strpos` which should be faster, but it's a just a matter of personal taste in this context.
Feel free to keep it unchanged.